test/unit/taxonomy_prototype/taxon_finder_test.rb

```diff
@@ -0,0 +1,24 @@
+require "test_helper"
+require "taxonomy_prototype/taxon_finder"
+
+describe TaxonomyPrototype::TaxonFinder do
+  describe "find_by(slug:)" do
+    before do
+      File.stubs(:exist?).returns(true)
```

Comment:
Is this block only relevant to the first test? If that's the case it might be clearer to inline it.